### ISSUE TITLE
Eclipse cannot infer type arguments for MappedIterable (in AndTest)

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#207]
Apparently, Eclipse IDE can not infer type arguments for `MappedIterable` in `AndTest`, imply an error in the project. I'm using Eclipse Neon.3 (4.6.3).

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by yegor256** [comment]
@fabriciofx can you provide a full error message?

**Comment by fabriciofx** [comment]
@yegor256 As you requested:

```
Cannot infer type arguments for MappedIterable<>	AndTest.java	/cactoos/src/test/java/org/cactoos/func	line 100
Cannot infer type arguments for ScalarHasValue<>	AndTest.java	/cactoos/src/test/java/org/cactoos/func	line 105
The constructor And(new MappedIterable<>(new ArrayAsIterable<>("hello", "world"), new ProcAsFunc<>(list::add, () -> true))) is undefined	AndTest.java	/cactoos/src/test/java/org/cactoos/func	line 99
The method assertThat(String, T, Matcher<? super T>) in the type MatcherAssert is not applicable for the arguments (String, And, new ScalarHasValue<>(Matchers.allOf(Matchers.equalTo(true), new FuncAsMatcher<>((<no type> value) -> (list.size() == 2)))))	AndTest.java	/cactoos/src/test/java/org/cactoos/func	line 97
```

**Comment by alex-semenyuk** [comment]
Seems this bug of Eclipse Neon https://bugs.eclipse.org/bugs/show_bug.cgi?id=497193 and related https://bugs.eclipse.org/bugs/show_bug.cgi?id=496579?

**Comment by fabriciofx** [comment]
@yegor256 Could you review the PR and if it's ok do a merge? Thanks in advance!

**Comment by fabriciofx** [comment]
@yegor256 Any comment about the PR #220 ?

**Comment by fabriciofx** [comment]
Closed #220.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/cactoos#207` was not in scope
